packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -144,8 +144,6 @@ library LibVerifying {
                 blk = state.blocks[slot];
                 if (blk.blockId != blockId) revert L1_BLOCK_MISMATCH();
 
-                tid = LibUtils.getTransitionId(state, blk, slot, blockHash);
```

Comment:
Why delete this? Before a block can be verified, you need to find the right trnasition, you need to look up the tid based the parent hash (`blockHash` in the code).

Reply:
Should we split LibVerifying into one more file called LibGenesis.sol and bring the init function there.

Reply:
Yep thanks for catching! 
Sorry was a test issue and i removed the wrong line ! (Fixed, commited soon)
I do not think we should split this file, but as you like!